### PR TITLE
feat: scan-googleapis-gen-and-create-pull-requests

### PR DESCRIPTION
**Pull request by SurferJeffAtGoogle**
Scans the googleapis/googleapis-gen repo, and creates copy-code pull requests.

### CODE REVIEW

File: packages/owl-bot/src/copy-code.ts [hunk 1]
```diff
@@ -196,6 +187,34 @@ export async function loadOwlBotYaml(destDir: string): Promise<OwlBotYaml> {
...
 
+/**
+ * Clones remote repos.  Returns local repos unchanged.
+ * @param repo a full repo name like googleapis/nodejs-vision, or a path to a local directory
+ * @param workDir a local directory where the cloned repo will be created
+ * @param logger a logger
+ * @param depth the depth param to pass to git clone.
+ * @returns the path to the local repo.
+ */
+export function toLocalRepo(
```
Comment (bcoe) [inline packages/owl-bot/src/copy-code.ts:198]: I like the introduction of this helper to encapsulate some of the git logic.

**Review by bcoe — COMMENTED** [review]
This logic looks good to me, I believe you said you were following on with some tests though?

File: packages/owl-bot/src/copy-code.ts (hunk 1 above)
Comment (SurferJeffAtGoogle) [reply packages/owl-bot/src/copy-code.ts:198]: > This logic looks good to me, I believe you said you were following on with some tests though?

Yes.  I added tests in https://github.com/googleapis/repo-automation-bots/pull/1488.  Let's continue the review there, and I'll close this PR.

**Comment by SurferJeffAtGoogle** [comment]
Obviated by #1488.

**Pull request closed without merging**